pkg/profileparser/profileparser.go

```diff
@@ -56,6 +60,179 @@ func GetPrefixedName(pbName, objName string) string {
 	return pbName + "-" + objName
 }
 
+func ParseBundle(contentDom *xmldom.Document, pb *cmpv1alpha1.ProfileBundle, pcfg *ParserConfig) error {
+	err := ParseProfilesAndDo(contentDom, pb, func(p *cmpv1alpha1.Profile) error {
+		err := parseAction(p, "Profile", pb, pcfg, func(found, updated interface{}) error {
```

Comment:
Honestly I dislike that the block that calls `parseAction` is a callback for the per-type parsers which itself has an update callback. This is all done so that the per-type payload can be updated. I tried working around this by adding UpdatePayload methods to the three types and passing an interface instead, but that seems a bit clunky as well...

Reply:
This part is fine with me.

Reply:
Thanks, marking as resolved, then